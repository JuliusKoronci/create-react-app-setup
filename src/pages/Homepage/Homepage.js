// @flow
import React from 'react';
import {Link} from 'react-router-dom';
import {getPath} from '../../routes';

const Homepage = () => (
    <div>
        Homepage
        <br />
        <Link to={getPath('dashboard')}>Dashboard</Link>
        <br />
        <Link to={getPath('demo', {id: 1}, {limit: 10})}>Demo</Link>
    </div>
);

export default Homepage;
