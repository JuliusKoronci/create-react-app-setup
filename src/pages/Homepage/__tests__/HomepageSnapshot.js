import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Homepage from '../Homepage';

const wrapper = shallow(<Homepage />);
describe('(Component) Homepage', () => {
  it('Should match snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});

