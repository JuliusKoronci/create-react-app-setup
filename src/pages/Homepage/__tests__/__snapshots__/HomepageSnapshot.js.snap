// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) Homepage Should match snapshot 1`] = `
<div>
  Homepage
  <br />
  <Link
    replace={false}
    to="/dashboard"
  >
    Dashboard
  </Link>
  <br />
  <Link
    replace={false}
    to="/demo/1?limit=10"
  >
    Demo
  </Link>
</div>
`;
